### PR TITLE
'About Me' additions and small edits to index and gallery

### PR DESCRIPTION
**Pull request by mivdittamo**
- Remove the line breaks from index.html
- Add a placeholder for gallery.html
- Add information and images to aboutMe.html

### CODE REVIEW

File: portfolio/src/main/webapp/aboutMe.html [hunk 1]
```diff
@@ -2,15 +2,27 @@
...
-    <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
-    <img src="images/gtLogoBuzz.png" width="200"/>
-    <p>At school, I am a Teaching Assistant in the College of Computing! I have previously TA'ed for the Introduction to Computing course.</p>
+    <div id="content">
+        <h1>About Me</h1>
+        <p>I am Isabelle Dittamo!</p>
+        <img src="images/profilePicture.jpg" height="300"/>
+        <p>I am an intern at Google as a part of the STEP Internship for the Summer of 2020!</p>
+        <img src="images/googleGLogo.jpg" width="200"/>
+        <p>I'm also a rising 3rd year student at the Georgia Institute of Technology studying Computer Science with concentrations in Information Internetworks and Intelligence.</p>
```
Comment (estep-google) [inline portfolio/src/main/webapp/aboutMe.html]: Please limit lines to a character limit of ~100 characters/line. This improves readability during review and later for subsequent code readers. Here and elsewhere.

I couldn't find a way to put a character indicator in the editor for Cloud Shell (standard option in an IDE).

When you do this, follow this form:

```
<p>
   Text ...
   More text ...
</p>
```

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,9 +10,9 @@
     <div id="content">
       <h1>Isabelle's Portfolio</h1>
       <p>
-          I'm Isabelle, and I'm a rising 3rd year student majoring in Computer Science at the
-          <br>Georgia Institute of Technology! Currently, I am interning at Google as a STEP Intern!
-          <br>This is my portfolio!
+          I'm Isabelle, and I'm a rising 3rd year student majoring in Computer Science at the
```
Comment (estep-google) [inline portfolio/src/main/webapp/index.html]: nit: trailing whitespace

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -2,8 +2,9 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  max-width: 60em;
```
Comment (estep-google) [inline portfolio/src/main/webapp/style.css:4]: nit: CSS best practice is to alphabetize properties

**Review by estep-google — COMMENTED** [review]
Just some small style comments

File: portfolio/src/main/webapp/aboutMe.html (hunk 1 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/aboutMe.html]: Got it! Fixed.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/index.html]: Fixed!

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (mivdittamo) [reply portfolio/src/main/webapp/style.css:4]: Fixed!

**Review by estep-google — APPROVED** [review]
lgtm

**Pull request merged**